docs/2.7.0/docs/canton/includes/snippet_data/tutorials/getting_started.json

```diff
@@ -1,27 +1,27 @@
 {
   "385" : {
     "command" : "val bank = participant2.parties.enable(\"Bank\", waitForDomain = DomainChoice.All)",
-    "output" : "bank : PartyId = Bank::12207f6b1097..."
```

Comment:
Is there any material change in this particular file? Or are the differences just an artifact of of running the code generation? In the latter case, can one exclude such files in a PR?

Reply:
Those look to be incidental. Reverted changes to all json files except for one